CMakeLists.txt

```diff
@@ -716,6 +723,8 @@ add_custom_target(run-benchmark
 
 set_target_properties(run-benchmark PROPERTIES EXCLUDE_FROM_ALL 1)
 
+if (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
```

Comment:
Everything under this is not re-indented, is this intentional?
Also, you have a comment at the end # End of Dieharder targets, would it be good to have one at the beginning?

Reply:
I'll change the closing comment to `# End of x64_64 only targets`